ONSdigital/census-rm-print-file-service
# Rmc 270 271 ce estab ni individual letter and pqs

# Motivation and Context

Added in the new Action Types and Pack Codes for NI CE Estab individual print files

# What has changed



- Added new mappings for Pack codes and action types

# How to test?



- Run `make build_and_test` and  all should pass
- Run with other branches and ATs should pass

# Links



[Trello](https://trello.com/c/aRah4j2n/841-ic05-ic06-ic08-generate-ce-estab-individual-addressed-print-files-ni-5)
# Screenshots (if appropriate):
